public/js/main.js

```diff
@@ -93,7 +93,7 @@ function setFileURLs(new_provider) {
           copyEl.children('ul').remove();
           copyEl.append('<ul class="dropdown-menu copy-options">' +
             '<li><a data-copy-type="https:" class="copy-https-url copy-button" href="javascript:void(0);">Copy Url</a></li>' + SRIcopyWithoutTagButton +
-              SRIcopyButton + 
+              SRIcopyButton +
```

Comment:
I realise this whitespace change is technically unrelated, but most IDEs will always remove this as trailing whitespace has no use. I think it best to be left in here so that we don't need to worry about it again.

Unless @PeterDaveHello you want to make a dedicated commit to master to remove this whitespace?